includes/class-wcpdf-admin.php

```diff
@@ -400,7 +400,7 @@ public function data_input_box_content ( $post ) {
 							<p class="form-field form-field-wide">
 								<p>
 									<span><strong><?php _e( 'Invoice Date:', 'woocommerce-pdf-invoices-packing-slips' ); ?></strong></span>
-									<span><?php if (!empty($invoice_date)) echo $invoice_date->date_i18n( wc_date_format().' @ '.wc_time_format() ); ?></span>
+									<span><?php if (!empty($invoice_date)) echo $invoice_date->date_i18n( wcpdf_date_format( $invoice, 'invoice_date' ).' @ '.wc_time_format() ); ?></span>
```

Comment:
This is perhaps not an ideal use case for this function because of the usage of `wc_time_format` here. It's backend so maybe we don't need it here?